.gitmodules

```diff
@@ -1,7 +1,9 @@
 [submodule "FV3"]
 	path = FV3
-	url = https://github.com/NOAA-EMC/fv3atm
-	branch = release/public-v1
+	#url = https://github.com/NOAA-EMC/fv3atm
+	#branch = release/public-v1
+	url = https://github.com/climbfuji/fv3atm
```

Comment:
@junwang-noaa we accidentally merged this temporary change to .gitmodules - need to fix and possibly correct submodule pointer for fv3atm.

Reply:
Please ignore. This was fixed in a follow-up PR.